docs/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Logs
```

Comment:
does this folder need a separate gitignore?

Reply:
Nope, just /build.

Reply:
Kept a ```.gitignore``` in ```docs``` so that the root package is completely separated from the ```docs``` package, except for ```.travis.yml```. Same reason why I have ```prettier``` and ```eslint``` duplicated in both.